editor/test/integration/material_test.clj

```diff
@@ -33,6 +33,24 @@
       (is (some? (g/node-value node-id :shader)))
       (is (= 1 (count samplers))))))
 
+(deftest material-pbr-parameters
+  ;; Test that all exposed PBR parameters are found, and that they are true
+  (test-util/with-loaded-project
+    (let [node-id (test-util/resource-node project "/materials/test_pbr_materials.material")
+          pbr-parameters (get-in (g/node-value node-id :build-targets) [0 :user-data :material-desc-with-build-resources :pbr-parameters])]
+      (is pbr-parameters
+         {:has-iridescene true
+          :has-metallic-roughness true
+          :has-volume true
+          :has-clearcoat true
+          :has-sheen true
+          :has-specular true
+          :has-transmission true
+          :has-specular-glossiness true
+          :has-ior true
+          :has-parameters true
+          :has-emissive-strength true}))))
+
```

Comment:
This needs to be`(is (= pbr-parameters {...}))` to actually test equality. Also, it should be in `(= expected actual)` order to get reported correctly when it fails. When writing a test, it's a good idea to make it fail on purpose just to make sure it will actually fail when something is off.